services/QuillLMS/db/migrate/20221109181742_create_teacher_info.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateTeacherInfo < ActiveRecord::Migration[6.1]
+  def change
+    create_table :teacher_infos do |t|
+      t.integer :minimum_grade_level
+      t.integer :maximum_grade_level
+      t.references :user, null: false, presence: true
```

Comment:
Do you mean `foreign_key: true` here instead of presence (which is for model files).

Also, I noticed a uniqueness validation in the model file.  To avoid [race conditions](https://thoughtbot.com/blog/validation-database-constraint-or-both), I would add a uniqueness constraint in the database too:

`add_index : teacher_infos, :user_id, unique: true`

Reply:
ooh, yes, I did! I'm surprised the migration ran with no issues for that.